src/algebra/parity.lean

```diff
@@ -119,13 +191,14 @@ end semiring
 section ring
 variables [ring α] {m n : α}
 
-@[simp] theorem even_neg (a : α) : even (-a) ↔ even a :=
-dvd_neg _ _
+@[simp] lemma even_neg_two : even (- 2 : α) := by simp only [even_neg, even_two]
 
-@[simp] lemma even_neg_two : even (- 2 : α) := by simp
-
-lemma even.sub_even (hm : even m) (hn : even n) : even (m - n) :=
-by { rw sub_eq_add_neg, exact hm.add_even ((even_neg n).mpr hn) }
+-- from src/algebra/order/ring.lean
```

Comment:
I don't feel this comment is really needed, people can just view the differences with `git log -p`.
```suggestion
```